.ci/SetupDevEnv/src/SetupDevEnv.jl

```diff
@@ -1,113 +1,659 @@
+"""
+The script checks the dependencies of the current project and provides a Julia environment that
+provides all current development versions of the QED dependencies.
+
+The following steps are carried out
+1. Create dependency graph of the active project without instantiating it. To do this, the QED
+project is cloned and the current dev branch is used.
+2. Calculate the correct order for all QED packages in which the packages must be added to the
+environment so that no QED package is added as an implicit dependency.
+3. Calculates which packages must be added in which order for the QED package to be tested.
+4. Remove all QED packages from the current environment to prevent QED packages defined as
+implicit dependencies from being instantiated.
+5. Install the QED package for testing and all QED dependencies in the correct order. If it is a
+dependency, the compat entry is also changed to match the QED package under test.
+
+The script needs to be executed the project space, which should be modified.
```

Comment:
```suggestion
The script checks the dependencies of the current project and provides a Julia environment that
provides all current development versions of the QED dependencies.

The following steps are carried out
1. Create a dependency graph of the active project without instantiating it. The QED
project is cloned to do this, and the current dev branch is used.
2. Calculate the correct order for all QED packages in which the packages must be added to the
environment so that no QED package is added as an implicit dependency.
3. Calculate which packages must be added in which order for the QED package to be tested.
4. Remove all QED packages from the current environment to prevent QED packages defined as
implicit dependencies from being instantiated.
5. Install the QED package for testing and all QED dependencies in the correct order. If it is a
dependency, the compat entry is also changed to match the QED package under test.

The script must be executed in the project space, which should be modified.
```